Documentation/install/azure/azure-terraform.md

```diff
@@ -103,8 +103,31 @@ Initialize the Tectonic Terraform modules.
 
 ```bash
 $ terraform init platforms/azure
+Downloading modules...
+Get: modules/bootstrap-ssh
+Get: modules/azure/resource-group
+Get: modules/azure/vnet
+Get: modules/azure/etcd
+Get: modules/ignition
+Get: modules/azure/master-as
+Get: modules/ignition
+Get: modules/azure/worker-as
+Get: modules/dns/azure
+Get: modules/tls/kube/self-signed
+Get: modules/tls/etcd
+Get: modules/tls/ingress/self-signed
+Get: modules/tls/identity/self-signed
+Get: modules/bootkube
+Get: modules/tectonic
+Get: modules/net/flannel-vxlan
+Get: modules/net/calico-network-policy
+
+Initializing provider plugins...
```

Comment:
my output, azure: 
```
...

Initializing provider plugins...
- Checking for available provider plugins on https://releases.hashicorp.com...
- Downloading plugin for provider "azurerm" (0.1.7)...

Terraform has been successfully initialized!
```

Do you want to show the provider plugin output for each platform as well? Or even ellipse to that key line "successfully initialized!", that lets the user know its over?